Source/ChocolateyGui/ViewModels/RemoteSourceViewModel.cs

```diff
@@ -276,7 +278,7 @@ protected override void OnInitialize()
                             p.IsInstalled = true;
                         }
 
-                        Packages.Add(p);
+                        Packages.Add(Mapper.Map<IPackageViewModel>(p));

```

Comment:
Huh, does this actually work? I didn't think Automapper likes interfaces.

Reply:
I'm not sure, it would need to be tested. I bet you are correct.

Reply:
It was wrong. Pushed fix

Reply:
Not quite fixed... it was using a package factory before.

Reply:
@chocolatey/chocolateygui can someone explain how this gets resolved by autofac? https://github.com/chocolatey/ChocolateyGUI/blob/c3ef83d27dd23fe2785435e4bebc7e291b454e44/Source/ChocolateyGui/Services/ChocolateyRemotePackageService.cs#L48